src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -1,12 +1,12 @@
 import _ from 'lodash/fp'
 import { Fragment, useEffect, useState } from 'react'
-import { div, h } from 'react-hyperscript-helpers'
+import { div, h, span } from 'react-hyperscript-helpers'
```

Comment:
If you accept all span removals on this page, for your batch commit convenience:
```suggestion
import { div, h } from 'react-hyperscript-helpers'
```